app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ContactsController < ApplicationController
+  def create
+    company = Company.find(params[:company_id])
+    @contact = company.contacts.new(contact_params)
+    @contact.save
```

Comment:
be sure to have some control flow here. what happens if the contact doesn't save? does a user see a nice error?